### PR TITLE
[Feature]: Share buttons

### PR DESCRIPTION
**Pull request by 1ucius**
Users can share to their social media using share buttons after swap or stake.

About #126

# How Has This Been Tested?

Chrome Desktop

# UI:

Buy G$ popup:
Screenshot_4

Screenshot_7

Screenshot_8

Stake:
Screenshot_9

Darkmode:
Screenshot_10

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [ ] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [x] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

**Comment by sirpy** [comment]
@L03TJ3 please do a code review

**Comment by YpetrovAJ** [comment]
@L03TJ3 can you merge it?

File: src/components/ShareTransaction/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
```
Comment (L03TJ3) [inline src/components/ShareTransaction/index.tsx:1]: Edit: you could maybe just import react? we don't need the whole library

File: src/components/Icon/Icon.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
```
Comment (L03TJ3) [inline src/components/Icon/Icon.tsx:1]: Same here, just import React

File: src/components/Share/index.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
+import * as React from 'react'
```
Comment (L03TJ3) [inline src/components/Share/index.tsx:1]: import React, { useState, useMemo } from 'react'

**Comment by 1ucius** [comment]
@L03TJ3 Fixed

**Pull request merged**